Sources/PointFree/Account/PrivateRss.swift

```diff
@@ -288,7 +288,7 @@ private func item(forUser user: User, episode: Episode) -> RssItem {
     enclosure: .init(
       length: episode.fullVideo.bytesLength,
       type: "video/mp4",
-      url: episode.fullVideo.downloadUrl
+      url: episode.fullVideo.downloadUrl(.hd720)
```

Comment:
We could potentially do 1080p for the latest episode...but maybe we wait till we figure out if our bandwidth gets in the acceptable range.